BankClient/Bank.cs

```diff
@@ -1,166 +1,129 @@
+    /// </summary>
+    /// <returns>Nothing</returns>
+    public void DeleteUserInformation(Guid id)
+    {
+        if (string.IsNullOrEmpty(id.ToString()) == false)
         {
-            Console.WriteLine("Nobody was found");
-            return;
+            Users.Remove(Users.Find(x => x.Id == id));
         }
-        Console.WriteLine($"Firstname: {filterlist[0].FirstName}, LastName: {filterlist[0].LastName}, Phonenumber: {filterlist[0].Phonenumber}, userID: {filterlist[0].ID}");
+        
     }
-
-    public void ShowAllUsers()
+    /// <summary>
+    /// Function prints list of users
+    /// </summary>
+    /// <returns>Nothing</returns>
+    public void PrintAllUsers()
     {
         if (Users != null)
         {
             int i = 1;
             foreach (var user in Users)
             {
                 Console.WriteLine(
-                    $"User {i}: Firstname: {user.FirstName}, LastName: {user.LastName}, Phonenumber: {user.Phonenumber}, userID: {user.ID}");
-                    i++;
+                    $"User {i}: Firstname: {user.FirstName}, LastName: {user.LastName}, PhoneNumber: {user.PhoneNumber}, userID: {user.Id}");
+                i++;
             }
         }
         else
         {
             Console.WriteLine("There are not users in the list");
         }
     }
-    public void ChangeUserInfomation()
+    
+    /// <summary>
+    /// Show "filterList" for method GetAllUserInformation <see cref="GetAllUsersInformation"/>
+    /// </summary>
+    private void ShowListOfUsers(List<UserInformation> listOfUsers)
     {
-        Console.Write("Enter userID: ");
-        var id = Console.ReadLine().Trim();
-        bool flag = false;
-        foreach (var user in Users)
+        foreach (var user in listOfUsers)
         {
-            if (user.ID.ToString() == id)
-            {
-                flag = true;
-                Console.Write("Firstname: ");
-                user.FirstName = Console.ReadLine();
-                Console.Write("Lastname: ");
-                user.LastName = Console.ReadLine();
-                Console.Write("Phonenumber: ");
-                user.Phonenumber = Console.ReadLine();
-                Console.WriteLine("The client was changed");
-                break;
-            }
-        }
-
-        if (flag == false)
-            Console.WriteLine("User wasn't found");
-    }
-
-    public void AddUserInformation()
-    {
-        Users.Add(new UserInformation());
-    }
-
-    public void DeleteUserInformation(List<UserInformation> filterlist)
-    {
-        if (filterlist.Count == 0)
-        {
-            Console.WriteLine("Nobody was found");
-            return;
-        }
-        foreach (var user in filterlist)
-        {
-            Users.RemoveAll(x => x.ID == user.ID);
+            Console.WriteLine(
+                $" Firstname: {user.FirstName}, LastName: {user.LastName}, PhoneNumber: {user.PhoneNumber}, userID: {user.Id}");
+         
```

Comment:
As I remember we decided to remove these methods from the repository and you need to leave it only in your local version for tests

Reply:
Yes, I know
I am not used to work with Git yet